### PR TITLE
ESQL: Move description of commands in docs

### PR DESCRIPTION
**Pull request by nik9000**
This copies the first line of the description of each command to just under the syntax so that it's "in order", before the `Parameters` section. That way if you are reading from top to bottom you see:
```
short description
syntax
parameter names and descriptions
long description
examples
```

I've also removed the `Description` section entirely if the description was just one sentence. So in some cases that just isn't `long description`.

### CODE REVIEW

File: docs/reference/esql/processing-commands/dissect.asciidoc [hunk 1]
```diff
@@ -56,4 +59,4 @@ include::{esql-specs}/docs.csv-spec[tag=dissectWithToDatetime]
 include::{esql-specs}/docs.csv-spec[tag=dissectWithToDatetime-result]
 |===
 
-// end::examples[]
\ No newline at end of file
+// end::examples[]
```
Comment (nik9000) [inline docs/reference/esql/processing-commands/dissect.asciidoc:62]: The text editor seems to have deleted some trailing newlines and trailing spaces. It makes this harder to read but I think it's probably worth having anyway.

**Comment by leemthompo** [comment]
Thanks @nik9000 — I'd actually put the `short description` before the `syntax`, I think that makes sense

**Comment by nik9000** [comment]
I'll give that a go.

**Comment by nik9000** [comment]
@leemthompo moved. Have another look?

**Review by leemthompo — APPROVED** [review]
Great improvement, thanks @nik9000 🚀

**Comment by nik9000** [comment]
Thanks @leemthompo !

**Pull request merged**